openvdb_points_houdini/houdini/SOP_NodeVDBPoints.h

```diff
@@ -140,125 +154,140 @@ class OPENVDB_HOUDINI_API SOP_NodeVDBPoints: public SOP_NodeVDB
+            if (gridName.isstring()) infoStr << " name: '" << gridName << "',";
 
-            typedef openvdb::tools::PointDataGrid PointDataGrid;
-            typedef openvdb::tools::PointDataTree PointDataTree;
-            typedef openvdb::tools::AttributeSet AttributeSet;
+            pointDataGridSpecificInfoText(infoStr, grid);
 
-            const PointDataGrid* pointDataGrid = dynamic_cast<const PointDataGrid*>(&grid);
+            infoStr << "\n";
 
-            if (!pointDataGrid)     continue;
+            gridn++;
+        }
 
-            const PointDataTree& pointDataTree = pointDataGrid->tree();
+        if (gridn > 0) {
+            std::ostringstream headStr;
+            headStr << gridn << " VDB point" << (gridn == 1 ? "" : "s") << "\n";
 
-            PointDataTree::LeafCIter iter = pointDataTree.cbeginLeaf();
+            parms.append(headStr.str().c_str());
+            parms.append(infoStr.str().c_str());
+        }
+    }
 
-            const openvdb::Index64 count = pointCount(pointDataTree);
+#endif // OPENVDB_LIBRARY_VERSION_NUMBER
 
-            infoStr_pts << " count: " << addDigitSeparators(count) << ",";
+    static void
+    pointDataGridSpecificInfoText(std::ostream& infoStr, const openvdb::GridBase& grid)
+    {
+        typedef openvdb::tools::PointDataGrid PointDataGrid;
+        typedef openvdb::tools::PointDataTree PointDataTree;
+        typedef openvdb::tools::AttributeSet AttributeSet;
 
-            if (!iter.getLeaf()) {
-                infoStr_pts << " attributes: ";
-                infoStr_pts << "<none>";
-            }
-            else {
-                const AttributeSet::DescriptorPtr& descriptor = iter->attributeSet().descriptorPtr();
+        const PointDataGrid* pointDataGrid = dynamic_cast<const PointDataGrid*>(&grid);
 
-                infoStr_pts << " groups: ";
+        if (!pointDataGrid) return;
 
-                const AttributeSet::Descriptor::NameToPosMap& groupMap = descriptor->groupMap();
+        // match native OpenVDB convention as much as possible
 
-                bool first = true;
-                for (AttributeSet::Descriptor::ConstIterator it = groupMap.begin(), it_end = groupMap.end();
-                        it != it_end; ++it) {
-                    if (first) {
-                        first = false;
-                    }
-                    else {
-                        infoStr_pts << ", ";
-                    }
+        infoStr << " voxel size: " << pointDataGrid->transform().voxelSize()[0] << ",";
 
-                    infoStr_pts << it->first << "(";
+        // rename ptdataidx32 to points for simplicity
 
-                    infoStr_pts << addDigitSeparators(groupPointCount(pointDataTree, it->first));
+        if (pointDataGrid->valueType() == "ptdataidx32")    infoStr << " type: points,";
+        else                                                infoStr << " type: "<< pointDataGrid->valueType() << ",";
```

Comment:
I guess it could just be hard-coded because the typedef PointDataTree refers to ptdataidx32, I was just thinking it could be easily extended to support ptdataidx64 for example. I'll change this.